_posts/2018-08-31-foreman-community-newsletter---august-2018.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: Foreman Community Newsletter - August 2018
+date: 2018-08-31 15:17
+author:
+  name: Greg Sutcliffe
+  uri: gwmngilfen
+tags:
+- foreman
+- newsletter
+- community
+excerpt: |
+  New releases! Oh, so many releases....
+---
+
+### Releases for everyone!
+
+We've had a lot of releases this month, so I'll group them all together. We've released:
+
+* [Foreman 1.19.0][1_19_0] was released on the very last day of August,
+  featuring support for Ubuntu Bionic, syslog logging, reporting failed builds,
+  and more ### Foreman 1.18.0 released
+* [1.17.3 and 1.17.4][1_17_4] bug fix releases. If you are still on 1.17, be
+  sure to upgrade to 1.17.4 before upgrading to 1.18!
+* [Katello 3.8 RC 2 & 3][k_3_8] - do give the RC a try and let us know how you find it :)
```

Comment:
https://github.com/theforeman/foreman-packaging/commit/473f1d9c4030112be8420f224d54359a83a9c9e0 was merged. Not sure if it's been officially released though.

Reply:
OK, fair enough. Now I need to go do nursery pickup - if the release is going out today, feel free to add it in.

Reply:
Today is labor day in the US so I don't think so. You could mention it's in the process of being released.

Reply:
Ok, I noted it in the release section, thanks